lib/interviewer/containers/Interfaces/Narrative.js

```diff
@@ -225,9 +230,6 @@ function mapStateToProps(state) {
   };
 }
 
-const mapDispatchToProps = () => ({
-});
+const mapDispatchToProps = () => ({});
```

Comment:
I _think_ you can remove this and omit the second parameter below

Reply:
resolved [3301d7b](https://github.com/complexdatacollective/Fresco/pull/77/commits/3301d7b56f5109648b4414804bc19c4c953307dc)